addon/templates/components/html-select.hbs

```diff
@@ -1,4 +1,4 @@
-<select {{action 'change' on='change'}} class="form-control">
+<select {{action 'change' on='change'}} class="{{mainComponent.elementClass}}">
```

Comment:
But just give me your opinion about this, if it would be better to default it in a forced default mode I would be still fine with this, but can imagine this could raise problems for some users.